tests/integration/helm_default_installation_test.go

```diff
@@ -177,7 +187,86 @@ func Test_Helm_Default_FluentD_Metadata(t *testing.T) {
 				tickDuration,
 			),
 		).
+		Assess("expected labels are present",
+			// TODO: refactor into a step func?
```

Comment:
As long as we're duplicating tests between OT and FluentD, I think we should always do this.

Reply:
Agreed, yet let's keep this "simple" (there's already going on in this PR :) )

Reply:
Ok, if you promise to do it later... :suspect: